### PR TITLE
[#36]  V9 Migration Schema Migration Tool (Liquibase) Documentation #36 

### LINKED ISSUE #36: V9 Migration Schema Migration Tool (Liquibase) Documentation
As part of https://github.com/TEAMMATES/teammates/issues/12048, documentation to use [Liquibase](https://contribute.liquibase.com/extensions-integrations/directory/integration-docs/gradle/) to migrate production schema.

Related to this https://github.com/TEAMMATES/teammates/pull/12930 which is the teammates dev process for liquibase

### PR DESCRIPTION
**Pull request by NicolasCwy**
Fixes #36 

**Outline of Solution**
Add documentation for release leader and PM on how to use Liquibase for schema changes such as adding and deleting columns/ tables

### CODE REVIEW

File: maintainer-guide.md [hunk 1]
```diff
@@ -152,6 +152,18 @@ Security vulnerabilities, once reported and confirmed, should be treated as a ca
 Since the detail of such vulnerability cannot be disclosed until it is fixed, an issue can be created just before a PR for the fix is submitted, with minimal information (e.g. simply "Security vulnerability" as an issue with no further description).
 The complete details can be filled in just before merging and/or after the fix is deployed.
 
+### Schema migration
+Schema migration is necessary when tables/ columns have to be added
```
Comment (ziqing26) [inline maintainer-guide.md:156]:
```suggestion
Schema migration is necessary when tables/columns are amended/added. Refer to TEAMMATES repo [docs/schema-migration.md](https://github.com/TEAMMATES/teammates/tree/master/docs/schema-migration.md)
```

**Review by ziqing26 — COMMENTED** [review]
Maybe can include the link to the doc in main repository (once the main repo PR is merged)

File: maintainer-guide.md [hunk 2]
```diff
@@ -152,6 +152,18 @@ Security vulnerabilities, once reported and confirmed, should be treated as a ca
 Since the detail of such vulnerability cannot be disclosed until it is fixed, an issue can be created just before a PR for the fix is submitted, with minimal information (e.g. simply "Security vulnerability" as an issue with no further description).
 The complete details can be filled in just before merging and/or after the fix is deployed.
 
+### Schema migration
+Schema migration is necessary when tables/ columns have to be added 
+
+**Role: RL**
+* Consolidate all changelogs in the current release folder in `src/main/resources/db/changelog` into a single changelog to put into `src/main/resources/db/changelog`
+* Follow dev guide on targeting changelog created and run `./gradlew liquibaseTag -PliquibaseCommandValue=<release-num>` to tag changelog to current release
```
Comment (ziqing26) [inline maintainer-guide.md:160]: @NicolasCwy Just want to clarify: Should RL or PM be the one running this command? As line 165 mentioned that PM also needs to run this

Comment (NicolasCwy) [reply maintainer-guide.md]: Amended this. RL has to add a changeset manually to tag the changelog. PM will run one command to execute changelog on DB

**Review by ziqing26 — APPROVED** [review]
LGTM!

**Review by cedricongjh — APPROVED** [review]
LGTM!

**Pull request merged**